### DISCUSSION TITLE
Where can I get the demo './examples/transmission_main_example.py' showed in 'https://telecominfraproject.github.io/oopt-gnpy/docs/images/transmission_main_example.svg'?

### DISCUSSION THREAD

**Discussion by panyupj** [Telecominfraproject/oopt-gnpy#473] [Q&A] [answered]
Where can I get the demo './examples/transmission_main_example.py' showed in 'https://telecominfraproject.github.io/oopt-gnpy/docs/images/transmission_main_example.svg'?

**Comment by jktjkt** [comment, marked as answer]
Sorry for confusion, it's a video that wasn't updated for some time. The correct information are mentioned in the docs, these days the script is called `gnpy-transmission-example` and it is installed into your `$PATH` during the installation (e.g., `pip install gnpy`).
